src/views/Home/index.tsx

```diff
@@ -29,26 +34,44 @@ const Banners = () => {
           </p>
         </div>
       )}
-      <SystemIntakeBanners />
+      <SystemIntakeBanners intakes={intakes} />
     </div>
   );
 };
 
 const Home = () => {
+  const dispatch = useDispatch();
   const { authState } = useOktaAuth();
   const userGroups = useSelector((state: AppState) => state.auth.groups);
   const isUserSet = useSelector((state: AppState) => state.auth.isUserSet);
+  const openIntakes = useSelector(
+    (state: AppState) => state.systemIntakes.openIntakes
+  );
+  const closedIntakes = useSelector(
+    (state: AppState) => state.systemIntakes.closedIntakes
+  );
+
+  useEffect(() => {
+    if (authState.isAuthenticated) {
+      dispatch(fetchSystemIntakes());
+    }
+  }, [dispatch, authState.isAuthenticated]);
 
   return (
     <PageWrapper>
       <Header />
       <MainContent className="grid-container margin-bottom-5">
         {isUserSet &&
           (user.isGrtReviewer(userGroups) ? (
-            <RequestRepository />
+            <RequestRepository
+              openIntakes={openIntakes.filter(
+                intake => intake.status !== 'INTAKE_DRAFT'
+              )}
+              closedIntakes={closedIntakes}
+            />
           ) : (
             <>
-              <Banners />
+              <Banners intakes={openIntakes} />
```

Comment:
If we only send open intakes for the banners, then is there any way for folks to see that their request has been withdrawn? Or, more importantly, to get to the task list for request that have had an LCID issued? I think we need to send all requests to the banners for now.